### PR TITLE
Minor grammar fixes for Guides

### PR DESCRIPTION
**Pull request by lindleyw**
Result of late night re-reading with blue pencil.

### CODE REVIEW

File: lib/Mojolicious/Guides/Cookbook.pod [hunk 1]
```diff
@@ -212,7 +212,7 @@ C<mod_proxy>, the configuration looks quite similar to the Nginx one above.
 =head2 Apache/CGI
 
 C<CGI> is supported out of the box and your L<Mojolicious> application will
-automatically detect that it is executed as a C<CGI> script. It's use in
+automatically detect that it is executed as a C<CGI> script. Its use in
```
Comment (jberger) [inline lib/Mojolicious/Guides/Cookbook.pod]: agree

File: lib/Mojolicious/Guides/Cookbook.pod [hunk 2]
```diff
@@ -452,7 +452,7 @@ nested closures that often result from continuation-passing style.
 
 =head2 Timers
 
-Another primary feature of the event loop are timers, which are created with
+Another primary feature of the event loop is timers, which are created with
```
Comment (jberger) [inline lib/Mojolicious/Guides/Cookbook.pod]: might be convinced

File: lib/Mojolicious/Guides/Rendering.pod [hunk 3]
```diff
@@ -1128,7 +1128,7 @@ applications, plugins make that very simple.
 
   1;
 
-The C<register> method will be called when you load the plugin and to add your
+The C<register> method will be called when you load the plugin; to add your
```
Comment (jberger) [inline lib/Mojolicious/Guides/Rendering.pod]: disagree

File: lib/Mojolicious/Guides/Rendering.pod [hunk 4]
```diff
@@ -443,8 +443,8 @@ L<Mojolicious::Plugin::DefaultHelpers/"accepts">.
 
 By now you've probably already encountered the built-in 404 (Not Found) and 500
 (Server Error) pages, that get rendered automatically when you make a mistake.
-Those are fallbacks for when your own exception handling fails, but especially
-during development they can also be a great help, you can render them manually
+Those are fallbacks for when your own exception handling fails. Especially
+during development, they can also be a great help: you can render them manually
```
Comment (jberger) [inline lib/Mojolicious/Guides/Rendering.pod]: Propose a third alternative

```
Those are fallbacks for when your own exception handling fails, which can be especially
helpful during development. You can also render them manually ...
```

that change would require rebalancing the lines of course

File: lib/Mojolicious/Guides/Rendering.pod (hunk 3 above)
Comment (Grinnz) [inline lib/Mojolicious/Guides/Rendering.pod]: I would agree with the change, these are two separate statements.

File: lib/Mojolicious/Guides/Rendering.pod (hunk 4 above)
Comment (Grinnz) [inline lib/Mojolicious/Guides/Rendering.pod]: +1 to jberger's suggestion

File: lib/Mojolicious/Guides/Cookbook.pod (hunk 2 above)
Comment (Grinnz) [inline lib/Mojolicious/Guides/Cookbook.pod]: This is a tricky one, "timers" is a single feature but "timers" is plural. Perhaps better to use timers as the subject:
    Timers, another primary feature of the event loop, are created with ...

File: lib/Mojolicious/Guides/Rendering.pod (hunk 3 above)
Comment (jberger) [inline lib/Mojolicious/Guides/Rendering.pod]: oh, crap, I actually agree, I seem to have gone red/green colorblind on this one. to be clear, I prefer the semicolon. sorry for the confusion

File: lib/Mojolicious/Guides/Cookbook.pod (hunk 2 above)
Comment (kraih) [inline lib/Mojolicious/Guides/Cookbook.pod]: What @Grinnz said. :+1:

File: lib/Mojolicious/Guides/Rendering.pod (hunk 4 above)
Comment (kraih) [inline lib/Mojolicious/Guides/Rendering.pod]: I like @jberger 's proposal too.

File: lib/Mojolicious/Guides/Rendering.pod (hunk 3 above)
Comment (kraih) [inline lib/Mojolicious/Guides/Rendering.pod]: Perhaps two sentences?

**Pull request closed without merging**